common/src/test/java/org/jboss/gm/common/ConfigurationTest.java

```diff
@@ -57,7 +59,7 @@ public void verifyEnv() {
 
         assertEquals(deploy, c.deployUrl());
 
-        assertEquals(c.restRetryDuration(), Translator.RETRY_DURATION_SEC);
+        assertEquals(Translator.RETRY_DURATION_SEC, c.restRetryDuration());
```

Comment:
Unrelated to this PR, but found the argument order was reversed.

Reply:
Ok